### ISSUE TITLE
"There's No Claim Here" but yet, "Your selected area overlaps an existing claim".

### ISSUE THREAD

**Issue by Generzon** [GriefPrevention/GriefPrevention#1775]
### Observed Behavior

I'm trying to unclaim someones claim, as an admin, but I get "theres no claim here" but if I try to claim it, I get "Your selected area overlaps an existing claim". so I'm not sure what's going wrong.

### Expected Behavior

There should either be a claim, or not be a claim.

### Reproduction steps

1. User claims area.
2. Admin runs /deleteclaim
3. Claim doesn't exist
4. But can't claim over

### Stack trace or error log

```shell
There is no error generated when doing this command.
```

### Server version

```shell
[26.01 21:35:02] [Server] [INFO] This server is running Paper version git-Paper-175 (MC: 1.18.1) (Implementing API version 1.18.1-R0.1-SNAPSHOT) (Git: 9490475)
[26.01 21:35:02] [Server] You are version(s) behind
[26.01 21:35:02] [Server] Download the new version athttps://papermc.io/downloads
```

### GriefPrevention version

```shell
[26.01 21:35:49] [Server] [INFO] GriefPrevention version 16.17.1-2b69191
```

### Configuration

```yaml
# Default values are perfect for most servers.  If you want to customize and have a question, look for the answer here first: http://dev.bukkit.org/bukkit-plugins/grief-prevention/pages/setup-and-configuration/

GriefPrevention:
  SeaLevelOverrides:
    world: -1
    world_nether: -1
    world_the_end: -1
    PvpWorld: -1
    parkourworld: -1
    flat_world: -1
    SurvivalWorld: -1
    season1world: -1
  Claims:
    Mode:
      parkourworld: Survival
      flat_world: Survival
      season1world: Survival
      world_nether: Survival
      world_the_end: Disabled
      world: Survival
      PvpWorld: Survival
      SurvivalWorld: Survival
    PreventGlobalMonsterEggs: true
    PreventTheft: true
    ProtectCreatures: true
    PreventButtonsSwitches: true
    LockWoodenDoors: false
    LockTrapDoors: false
    LockFenceGates: true
    EnderPearlsRequireAccessTrust: true
    RaidTriggersRequireBuildTrust: true
    ProtectHorses: true
    ProtectDonkeys: true
    ProtectLlamas: true
    InitialBlocks: 100
    Claim Blocks Accrued Per Hour:
      Default: 250
    Max Accrued Claim Blocks:
      Default: 50000
    Accrued Idle Threshold: 0
    AccruedIdlePercent: 0
    AbandonReturnRatio: 1.0
    AutomaticNewPlayerClaimsRadius: 0
    AutomaticNewPlayerClaimsRadiusMinimum: 0
    ExtendIntoGroundDistance: 5
    MinimumWidth: 5
    MinimumArea: 100
    MaximumDepth: 0
    InvestigationTool: STICK
    ModificationTool: GOLDEN_SHOVEL
    Expiration:
      ChestClaimDays: 14
      UnusedClaimDays: 14
      AllClaims:
        DaysInactive: 14
        ExceptWhenOwnerHasTotalClaimBlocks: 100000
        ExceptWhenOwnerHasBonusClaimBlocks: 10000
      AutomaticNatureRestoration:
        SurvivalWorlds: false
    AllowTrappedInAdminClaims: false
    MaximumNumberOfClaimsPerPlayer: 0
    CreationRequiresWorldGuardBuildPermission: true
    VillagerTradingRequiresPermission: true
    CommandsRequiringAccessTrust: /sethome
    DeliverManuals: true
    ManualDeliveryDelaySeconds: 30
    RavagersBreakBlocks: true
    FireSpreadsInClaims: false
    FireDamagesInClaims: false
    LecternReadingRequiresAccessTrust: true
  Spam:
    Enabled: false
    LoginCooldownSeconds: 60
    LoginLogoutNotificationsPerMinute: 5
    ChatSlashCommands: /me;/global;/local
    WhisperSlashCommands: /tell;/pm;/r;/whisper;/msg
    WarningMessage: Please reduce your noise level.  Spammers will be banned.
    BanOffenders: true
    BanMessage: Banned for spam.
    AllowedIpAddresses: 1.2.3.4; 5.6.7.8
    DeathMessageCooldownSeconds: 120
    Logout Message Delay In Seconds: 0
  PvP:
    RulesEnabledInWorld:
      world: true
      world_nether: true
      world_the_end: true
      PvpWorld: true
      parkourworld: true
      flat_world: true
      SurvivalWorld: true
      season1world: true
    ProtectFreshSpawns: true
    PunishLogout: true
    CombatTimeoutSeconds: 15
    AllowCombatItemDrop: false
    BlockedSlashCommands: /home;/vanish;/spawn;/tpa;/pvp off
    ProtectPlayersInLandClaims:
      PlayerOwnedClaims: true
      AdministrativeClaims: true
      AdministrativeSubdivisions: true
    AllowLavaDumpingNearOtherPlayers:
      PvPWorlds: false
      NonPvPWorlds: false
    AllowFlintAndSteelNearOtherPlayers:
      PvPWorlds: false
      NonPvPWorlds: false
    ProtectPetsOutsideLandClaims: true
  Economy:
    ClaimBlocksMaxBonus: 0
    ClaimBlocksPurchaseCost: 0.0
    ClaimBlocksSellValue: 0.0
  ProtectItemsDroppedOnDeath:
    PvPWorlds: false
    NonPvPWorlds: false
  BlockLandClaimExplosions: true
  BlockSurfaceCreeperExplosions: true
  BlockSurfaceOtherExplosions: true
  LimitSkyTrees: true
  LimitTreeGrowth: false
  PistonMovement: CLAIMS_ONLY
  PistonExplosionSound: true
  FireSpreads: false
  FireDestroys: false
  AdminsGetWhispers: true
  AdminsGetSignNotifications: true
  SmartBan: true
  Mute New Players Using Banned Words: true
  MaxPlayersPerIpAddress: 2
  Siege:
    Worlds: []
    BreakableBlocks:
    - GRASS_BLOCK
    - DIRT
    - COBBLESTONE
    - OAK_PLANKS
    - SPRUCE_PLANKS
    - BIRCH_PLANKS
    - JUNGLE_PLANKS
    - ACACIA_PLANKS
    - DARK_OAK_PLANKS
    - SAND
    - GRAVEL
    - GLASS
    - GRASS
    - FERN
    - DEAD_BUSH
    - WHITE_WOOL
    - ORANGE_WOOL
    - MAGENTA_WOOL
    - LIGHT_BLUE_WOOL
    - YELLOW_WOOL
    - LIME_WOOL
    - PINK_WOOL
    - GRAY_WOOL
    - LIGHT_GRAY_WOOL
    - CYAN_WOOL
    - PURPLE_WOOL
    - BLUE_WOOL
    - BROWN_WOOL
    - GREEN_WOOL
    - RED_WOOL
    - BLACK_WOOL
    - SNOW
    - GLASS_PANE
    DoorsOpenDelayInSeconds: 300
    CooldownEndInMinutes: 60
  EndermenMoveBlocks: false
  SilverfishBreakBlocks: false
  CreaturesTrampleCrops: false
  RabbitsEatCrops: true
  HardModeZombiesBreakDoors: false
  Database:
    URL: ''
    UserName: ''
    Password: ''
  UseBanCommand: false
  BanCommandPattern: ban %name% %reason%
  Advanced:
    fixNegativeClaimblockAmounts: true
    ClaimExpirationCheckRate: 60
    OfflinePlayer_cache_days: 90
  Abridged Logs:
    Days To Keep: 7
    Included Entry Types:
      Social Activity: true
      Suspicious Activity: true
      Administrative Activity: false
      Debug: false
      Muted Chat Messages: false
```

### Plugin list

```shell
[26.01 21:36:46] [Server] [INFO] Plugins (48): AureliumSkills, Bin*, BisectHosting, BuycraftX, ChatInjector*, ChatManager, Citizens, CitizensCMD, CoreProtect, DeluxeTags*, Essentials, EssentialsSpawn, floodgate, Geyser-Spigot, Graves, GriefPrevention, GSit, HeadsPlus, HolographicDisplays, LuckPerms, Marriage, Minepacks, Multiverse-Core, MyCommand, MyRules, NerfPhantoms, NoLavaKill, OnePlayerSleep*, Orebfuscator, PlaceholderAPI, PlayerHeads, PlayerParticles, Playtimes*, PlugMan, ProtocolLib, PvPToggle, RealScoreboard, SafeTrade, spark, StackMob, TAB, Themis, TradeShop, Vault, VeinMiner, ViaVersion, WorldEdit, WorldGuard
```

### Running without GriefPrevention

- [ ] I attempted running the server without GriefPrevention installed.
- [ ] The problem does _not_ occur when GriefPrevention is removed from the server.

### Running with only GriefPrevention

- [ ] I attempted running only GriefPrevention on the server.
- [ ] The issue still occurs when GriefPrevention is the only plugin running.

### Running on a fresh, clean server installation

- [ ] I attempted testing for the issue on a new server.
- [ ] The issue still occurs on a new server.

### Using unmodified client

- [ ] I attempted testing for the issue with the vanilla client.
- [ ] The issue still occurs when using the vanilla client.

### We appreciate you taking the time to fill out a bug report!

- [X] I searched for similar issues before submitting this bug report.

**Comment by Jikoo** [comment]
Please reproduce on a more recent GP build - there have been numerous fixes since the 16.17.1 release. 16.17.1 also has several holes in protections for 1.17/1.18 materials.

**Comment by Jikoo** [comment]
Closing as this was likely fixed by #1260 - if you can still reproduce on a more recent version of GP please provide the new version and I'll reopen it.
